src/registerServiceWorker.js

```diff
@@ -5,7 +5,9 @@ import { register } from 'register-service-worker';
 if (process.env.NODE_ENV === 'production') {
   register(`${process.env.BASE_URL}service-worker.js`, {
     ready() {
-      console.log('App is being served from cache by a service worker.\nFor more details, visit https://goo.gl/AFskqB');
+      console.log(
```

Comment:
Yes. but we could ignore this file if we want to. On other cases, the code isn't readable with 200 width. Such lines would require to scroll through horizontally. And again, hard to read through attributes and stuff. What do you think?

Reply:
I am updating with an above-mentioned rule but I still think 100 width is good to maintain readability. And I propose to add this file to `.prettierignore`. Because this is the only file we have this problem with. What do you think?

Reply:
I generally use 120 but it doesn't really matter

Reply:
Ah. I think our preferences come from our screen sizes. Do you happen to have a really big screen? Just curious.

Reply:
I have a 24" desktop but my laptop is 15.6". Are you on 13"?

Reply:
No, I've got the same size monitors too. Hmm. Anyway. 

Ah, I use a bit of zoom on my editor. Maybe that's the reason 🙂